src/core/lib/surface/call.cc

```diff
@@ -637,11 +637,10 @@ static void execute_batch_in_call_combiner(void* arg, grpc_error* /*ignored*/) {
 
 // start_batch_closure points to a caller-allocated closure to be used
 // for entering the call combiner.
-static void execute_batch(grpc_call* call,
-                          grpc_transport_stream_op_batch* batch,
+static void execute_batch(grpc_call* call, grpc_transport_stream_op_batch* op,
```

Comment:
The parameter here should be called `batch`, not `op`.  Let's fix the declaration instead.